lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
I too vote for ensuring that we put everything back the way it was.  (I'd actually vote for a default of "put it back the way it was" and a debugging option for "leave everything in the state it was in when the build failed", but that's beyond scope.)